src/feature_types/feature.js

```diff
@@ -55,4 +50,4 @@ Feature.prototype.internal = function(mode) {
   };
 };
 
-module.exports = Feature;
+export default Feature;
```

Comment:
This makes sense to me. It's for browser usage so will be compiled anyway, so we may as well use all ES2015. 

I like that you were able to drop the `ctx` from these features.